Examples/Scripts/Python/geomodel.py

```
@@ -115,74 +145,82 @@ def main():
+        print("Found ", len(materialSurfaces), " material surfaces")
+
+        # Output the detector to SVG
+        if args.output_svg:
+            surfaceStyle = acts.svg.Style()
+            surfaceStyle.fillColor = [5, 150, 245]
+            surfaceStyle.fillOpacity = 0.5
+
+            surfaceOptions = acts.svg.SurfaceOptions()
+            surfaceOptions.style = surfaceStyle
+
+            viewRange = acts.Extent([])
```

Ruff (0.8.0)

Local variable `viewRange` is assigned to but never used

Remove assignment to unused variable `viewRange`

(F841)